# Bumper values
uint8 LEFT_SIDE = 0         # left front side bumper switch
uint8 RIGHT_SIDE = 1        # right front side bumper switch
uint8 LEFT_FRONT = 2        # Left front bumper switch
uint8 RIGHT_FRONT = 3       # Right front bumper switch
uint8 LEFT_DROP = 4         # Left optical sensor indicating drop off
uint8 RIGHT_DROP = 5        # Right optical sensor indicating drop off
uint8 LEFT_MAG = 6          # Left magnet sensor
uint8 RIGHT_MAG = 7         # Right magnet sensor

uint8   bumper       # values defined above
bool    engaged      # whether or not the bumper is engaged (detecting something) or not